web-client/src/presenter/actions/TrialSession/computeTrialSessionFormDataAction.js

```diff
@@ -0,0 +1,80 @@
+import { state } from 'cerebral';
```

Comment:
@rachaelparris can you please look into this?

Reply:
This was being handled by leaving the term undefined, but I made a change to make that more explicit. Lines 59-77 of `computeTrialSessionFormDataAction.test.js` cover invalid months. It shouldn't throw an error, it just leaves the term undefined, and because that's a required field, the user will get a validation error when submitting.

Reply:
OK!